music/demos/visualizer.ts

```diff
@@ -70,8 +75,13 @@ function initPlayerAndVisualizer(seq: mm.INoteSequence) {
   playBtn.disabled = false;
   playBtn.textContent = 'Loading';
 
-  canvasVisualizer = new mm.PianoRollCanvasVisualizer(seq, canvas);
-  svgVisualizer = new mm.PianoRollSVGVisualizer(seq, svg);
+  let config = {
```

Comment:
is this needed? does everything look ok if it's not set?

Reply:
Currently under revision. Let's review this code when it is done in next PR.

Reply:
I loaded Aladdin with these settings and it looks pretty bad, so I don't think we want to keep them like this:
<img width="907" alt="Screen Shot 2019-04-12 at 11 01 31 AM" src="https://user-images.githubusercontent.com/1369170/56056936-7548a380-5d12-11e9-8026-7e84956b8d15.png">

Reply:
It has been revised and it is Ok. The code I guess you miss (the removed lines 73 and 74, doing the object construction) are a few lines below (82, 83 and 84). Actually they have not been translated, but merger algorithm did not match them properly. They moved below precisely due to the addition of a config.pixelsPerTimeStep setup to 60 pixels/second, because default value of 30px is too narrow to make room for note head width.

Regarding to the ugly aspect, I do agree such a fast score with lots of 1/64th, several tracks and just 60px per second width resolution looks horrible in proportional mode. It will improve if I force the visualizer to draw in compact mode on this first part of the demo (all visualizers following same player).  I can make this configuration change to 0px/second, which is the value to make it compact visualization, and apply it only to the mm.ScoreSVGVisualizer instance (the rest will keep default values as always).

I did it so because I wanted to show how well they matched (horizontally) checking the compatibility among 3 visualizers, but I can move it to a separate demo as you suggested.